site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -157,3 +152,69 @@ automatically for you:
             ...
         }
     }
+
+.. _nested-log:
+
+Nested log
+----------
+
+When you retry a failed attempt, you might want to record the result of each attempt and to group them under
+a single RequestLog_. A RequestLog_ can contain more than one child RequestLog_ in Armeria to support this sort
+of use cases.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.common.logging.RequestLogBuilder;
+
+    RequestLogBuilder.addChild(RequestLog);
+
+If the added RequestLog_ is the first child, the request-side log of the `RequestLog`_ will be propagated to the
+parent log. You can add as many children logs as you want, but the rest of logs would not effect.
+If you want to fill the response-side log of the parent log, please invoke:
+
+.. code-block:: java
+
+    RequestLogBuilder.endResponseWithLastChild();
+
+This will propagate the response-side log of the last added child to the parent log. So the parent log will be:
```

Comment:
So the parent log.. -> The following diagram illustrates how a RequestLog_ with children looks like: